src/errors.rs

```diff
@@ -73,6 +93,15 @@ macro_rules! verify_err {
     }};
 }
 
+macro_rules! arg_err {
+    ($x:expr) => {{
+        Err(crate::errors::InternalError::InvalidArgument(String::from(
+            $x,
+        )))
+    }};
+}
+
+#[allow(unused)]
```

Comment:
If these are not used anymore we should just delete them. I'd love to eventually have no macros on this file!